streams/src/test/java/org/apache/kafka/streams/state/internals/WindowBytesStoreTest.java

```diff
@@ -900,13 +827,13 @@ private void shouldLogAndMeasureExpiredRecords(final String builtInMetricsVersio
         streamsConfig.setProperty(StreamsConfig.BUILT_IN_METRICS_VERSION_CONFIG, builtInMetricsVersion);
         final WindowStore<Integer, String> windowStore =
             buildWindowStore(RETENTION_PERIOD, WINDOW_SIZE, false, Serdes.Integer(), Serdes.String());
-        final RecordCollector recordCollector = createRecordCollector(windowStore.name());
         recordCollector.init(producer);
         final InternalMockProcessorContext context = new InternalMockProcessorContext(
             TestUtils.tempDirectory(),
             new StreamsConfig(streamsConfig),
             recordCollector
         );
+        context.setTime(1L);
```

Comment:
With the MockRecordCollector we need to set timestamp as > 0 since otherwise the default -1 timetamp would not be accepted by ProduceRecord constructor. Ditto elsewhere.